client/system/info_linux.go

```diff
@@ -89,9 +90,17 @@ func _getInfo() string {
 func sysInfo() (serialNumber string, productName string, manufacturer string) {
 	var si sysinfo.SysInfo
 	si.GetSysInfo()
+	isascii, _ := regexp.Compile("^[[:ascii:]]+$")
```

Comment:
Please handle the error or use the regexp.Mustcompile

Reply:
fixed via regexp.Mustcompile